stable/mongodb-replicaset/tests/mongodb-up-test.sh

```diff
@@ -1,12 +1,43 @@
 #!/usr/bin/env bash
 
```

Comment:
Might be helpful to `set -ex` here so we can see the commands being ran. And it's a test so should fail fast if any error (not sure why it wasn't before).